src/js/components/settings/user-management/selfusermanagement.js

```diff
@@ -105,6 +119,10 @@ export const SelfUserManagement = ({ canHave2FA, currentUser, editUser, hasTrack
             </Form>
           </>
         ))}
+      <div className="clickable flexbox space-between margin-top" onClick={toggleMode}>
```

Comment:
We can comment this out for now to hide the button, and make the final dark theme adjustments in a future PR